chapters/introduction.tex

```diff
@@ -22,6 +22,9 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 verified for those classes, before using them to build the simulation
 model.
 
+The restrictions for a simulation model apply when simulating the model.
+Thus a tool may translate a model lacking some key parts -- provided they are given when simulating the model.
+
```

Comment:
This belongs to a different issue, right?

Reply:
Yes, removed.